src/OrchardCore.Modules/OrchardCore.Email.Azure/Drivers/AzureEmailSettingsDisplayDriver.cs

```diff
@@ -0,0 +1,157 @@
+        .OnGroup(EmailSettings.GroupId);
+    }
+
+    public override async Task<IDisplayResult> UpdateAsync(ISite site, AzureEmailSettings settings, IUpdateModel updater, BuildEditorContext context)
+    {
+        var user = _httpContextAccessor.HttpContext?.User;
+
+        if (!await _authorizationService.AuthorizeAsync(user, Permissions.ManageEmailSettings))
+        {
+            return null;
+        }
+
+        var model = new AzureEmailSettingsViewModel();
+
+        if (await updater.TryUpdateModelAsync(model, Prefix))
+        {
+            var emailSettings = site.As<EmailSettings>();
+
+            var hasChanges = model.IsEnabled != settings.IsEnabled;
+
+            if (!model.IsEnabled)
+            {
+                if (hasChanges && emailSettings.DefaultProviderName == AzureEmailProvider.TechnicalName)
+                {
+                    emailSettings.DefaultProviderName = null;
+
+                    site.Put(emailSettings);
+
+                    await _notifier.WarningAsync(H["You have successfully disabled the default Email provider. The Email service is now disable and will remain disabled until you designate a new default provider."]);
+                }
+
+                settings.IsEnabled = false;
+            }
+            else
+            {
+                settings.IsEnabled = true;
+
+                hasChanges |= model.DefaultSender != settings.DefaultSender;
+
+                settings.DefaultSender = model.DefaultSender;
+                var azureEmailOptions = _shellConfiguration.GetSection(AzureEmailOptionsConfiguration.SectionName).Get<AzureEmailOptions>();
+
+                if (azureEmailOptions is null || !azureEmailOptions.PreventUIConnectionChange)
+                {
+                    if (string.IsNullOrWhiteSpace(model.ConnectionString) && settings.ConnectionString is null)
+                    {
+                        context.Updater.ModelState.AddModelError(Prefix, nameof(model.ConnectionString), S["Connection string is required."]);
+                    }
+                    else if (!string.IsNullOrWhiteSpace(model.ConnectionString))
+                    {
+                        // Encrypt the connection string.
+                        var protector = _dataProtectionProvider.CreateProtector(AzureEmailOptionsConfiguration.ProtectorName);
+
+                        var protectedConnection = protector.Protect(model.ConnectionString);
+
+                        // Check if the connection string changed before setting it.
+                        hasChanges |= protectedConnection != settings.ConnectionString;
+
+                        settings.ConnectionString = protectedConnection;
+                    }
+                }
+            }
+
+            // Release the tenant to apply the settings when something changed.
+            if (context.Updater.ModelState.IsValid && hasChanges)
+            {
+                await _shellHost.ReleaseShellContextAsync(_shellSettings);
```

Comment:
why should a nontechnical user see this or case about seeing this kinds of warning? The app should do what the app needs without having to worry about explaining that to the user.

Reply:
I'll await your comment. But I kinda want to remove the warning from under the SMTP tab as well. No need for this kind of warning "specially, if nothing change, we do not actually reload".

Reply:
No nontechnical user will configure SMTP or ACS settings, that's already quite a high barrier. I'd also add to those messages to actually emphasize the point: "The current tenant will be reloaded when the settings are saved. This might cause several seconds when the app will be unresponsive."

Reply:
this only make since if you own the SAAS environment. If you are leasing a site, you don't know or care about tenant reload. Tenant reload should not be mentioned

"The current tenant will be reloaded when the settings are saved. This might cause several seconds when the app will be unresponsive."

But if you want a warning, I would add something like this instead. thoughts?

> The website might be restarted upon saving the settings, potentially leading to temporary unresponsiveness during the process.

Reply:
Sure, that's good. Then please do a search and replace for this because there are many instances of the same warning.

Reply:
Done.